### PR TITLE
Add utilities for sending investigations to D&B

### PR DESCRIPTION
**Pull request by alixedi**

### CODE REVIEW

File: company/utils.py [hunk 1]
```diff
@@ -82,3 +82,74 @@ def send_change_request_batch(change_requests, batch_identifier):
...
+        'Domain': company_details.get('domain'),
+        'Telephone Number': company_details.get('telephone_number'),
+        'DUNS Number': '',
+    }
+
+
+def generate_investigation_request_csv(investigation_requests):
+    """
+    Given a list of InvestigationRequest records, generate a CSV which is readable by D&B
+    support staff. The returned CSV content is a BytesIO file object, ready for sending by email.
```
Comment (currycoder) [inline company/utils.py]: :eyes: Copy paste!

The returned CSV content is a StringIO.

File: company/utils.py [hunk 2]
```diff
@@ -82,3 +82,74 @@ def send_change_request_batch(change_requests, batch_identifier):
...
+
+
+def send_investigation_request_batch(investigation_requests, batch_identifier):
+    """
+    Send a batch of investigation requests in a CSV by email to
+    settings.INVESTIGATION_REQUESTS_RECIPIENTS.
+    """
+    context = {
+        'batch_identifier': batch_identifier,
+        'link_to_file': _convert_stringio_to_bytesio(
```
Comment (currycoder) [inline company/utils.py]: I like this approach of making the CSV generation function deal in stringio and converting it when we need to into bytesio :ok_hand:

File: company/utils.py [hunk 3]
```diff
@@ -82,3 +82,74 @@ def send_change_request_batch(change_requests, batch_identifier):
...
+            generate_investigation_request_csv(investigation_requests),
+        ),
+    }
+
+    for email_address in settings.INVESTIGATION_REQUESTS_RECIPIENTS:
```
Comment (currycoder) [inline company/utils.py]: I believe that accessing `settings.INVESTIGATION_REQUESTS_RECIPIENTS` will result in an attribute error as I can't see it defined in the settings file.

File: company/utils.py (hunk 1 above)
Comment (alixedi) [reply company/utils.py]: Fixed in c6d73d47aa3e8fd505e5ee1475f6946534173432

File: company/utils.py (hunk 3 above)
Comment (alixedi) [reply company/utils.py:147]: Forgot to commit this. Fixed now 20d4f0bab4a69e9729d6db04c26bc177f317797b

**Pull request merged**